docs/src/getting-started/forc_project.md

```diff
@@ -39,7 +31,7 @@ Here are the contents of the only Sway file in the project, and the main entry p
 script;
 
 fn main() {
-    
```

Comment:
Woops, my editor's setup up to remove trailing whitespace automatically when writing the buffer